docs/content/en/docs/tasks/cluster/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -16,12 +16,24 @@ The upgrade command also upgrades core components of EKS Anywhere and lets the u
 >**_NOTE:_** Currently only Minor Version Upgrades are support for Bare Metal clusters. No other aspects of the cluster upgrades are currently supported.
 >
 
+### Modular Cluster Upgrades
+
+To allow users to be explicit when deciding which clusters to upgrade, workload clusters are not affected by management cluster upgrades.
+Therefore, each cluster must be individually upgraded to enjoy the latest features on all clusters.
+
+When triggering a workload cluster upgrade after upgrading the management cluster, it will not only reflect the changes in the cluster spec, but also any changes derived from the difference in business logic between the old and new EKS-A cluster controllers.
```

Comment:
Trying to make this info a bit more accessible to readers with different backgrounds:
```suggestion
When triggering a workload cluster upgrade after upgrading the management cluster, it will not only reflect your changes in the cluster spec, but also any new improvements included in the new EKS-A controller.
```